TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/BasicAuto.java

```diff
@@ -0,0 +1,56 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.hardware.Drivebase;
+import org.firstinspires.ftc.teamcode.hardware.VisionCamera;
+import org.firstinspires.ftc.teamcode.utility.FieldSide;
+
+import java.util.Objects;
+
+@Autonomous(name = "BasicAuto")
```

Comment:
You should include a header block comment here describing what the autonomous is intended to do.

Reply:
This is more of a "Testing Ground" auto, so should we add that in the comments?